Odata-docs/TOC.yml

```diff
@@ -167,6 +167,8 @@
     items:
     - name: Getting started
       href: /odata/webapi/netcore
+    - name: Build web APIs with OData
```

Comment:
How is this different from Getting Started document above? Can rename to a title like "Web  API with query options". Also  "with OData" seems unnecessary.